app/namespace.go

```diff
@@ -929,6 +929,17 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 						if n.Spec.AuthMethod == authMethod {
 							return errors.New("nothing to change")
 						}
+						if disruptiveChange(n.Spec.AuthMethod, authMethod) {
+							yes, err := ConfirmPrompt(ctx,
+								fmt.Sprintf("setting auth method to '%s' from '%s' will cause existing client connections to be dropped. "+
```

Comment:
Could be better more specific (drop sounds temporary) like `will cause existing client connections to fail authentication`.

Reply:
true...though the endpoint gets deleted, so client it's more of a connectivity issue rather than auth.